src/util/isEventEmitter.ts

```diff
@@ -0,0 +1,15 @@
+export type EventEmmitterLike1 = {
```

Comment:
No. This is only used in `fromEvent`. We can encapsulate this.

...But I intended to add it as a generic utility. Is this a premature optimization?